src/Providers/ZeroRPCContextProvider.php

```diff
@@ -46,6 +46,17 @@ public function register()
                 $middleware->beforeSendRequest()
             );
 
+            if (config('zerorpc.debugbar_rpc_monitor') === true) {
```

Comment:
If it possible to check if debugbar is installed/enabled and use that instead of this config?